app/invocation/invocation_not_found.tsx

```diff
@@ -37,7 +32,33 @@ export default class InvocationNotFoundComponent extends React.Component {
               </div>
             </div>
           </div>
-        )}
+        </div>
+      );
+    }
+
+    return (
+      <div className="state-page">
+        <div className="shelf">
+          <div className="container">
+            <div className="breadcrumbs">Invocation {this.props.invocationId}</div>
+            {this.props.error?.code === "NotFound" && (
```

Comment:
It's not really needed in TS, because the type of `error.code` is `type ErrorCode = 'NotFound' | 'PermissionDenied' | ...`. With string literal types like this, TS gives you all the same benefits as if you used a constant. e.g. if you try to check `error.code === 'SomeCodeThatDoesntExist'` then the build fails